### DISCUSSION TITLE
CSS background-image works fine on local but doesnt work on github.

### DISCUSSION THREAD

**Discussion by Stockton-slap** [community/community#55030] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Here is my repo https://github.com/Stockton-slap/macarons. 
If you check the network section, it shows 404 error, but on my local it works just fine and shows correct paths. Does anyone have any idea what i am doing wrong?

**Comment by emilburganov** [comment]
I ran your project locally, also 404 errors. So the problem is not in GitHub pages. Your css path is wrong. Change ../../ to ../

Eg:
../../images/phone.png
on
../images/phone.png
